docs/install-1.6.md

```diff
@@ -0,0 +1,164 @@
+# Installing Service Catalog on Clusters Running Kubernetes 1.7 and Above
```

Comment:
Is the "1.6 and Below" aspect true? Does the service catalog currently work on versions below 1.6? I would call a hard line at 1.6 just to avoid any issues.

Reply:
I changed to just 1.6 - you're right that 1.5 is a no-go.